src/lib/core/SymbolSolver.cpp

```diff
@@ -493,86 +512,99 @@ void SymbolSolver::solveNames(LinuxProcMapEntry * procMapEntry)
 			lst.push_back(&it->second);
 		}
 	}
+	if ( addr2lineCmd.str().length() > prefixLen) {
+	  //hide error if silent
+	  if (gblOptions != NULL && gblOptions->outputVerbosity <= MALT_VERBOSITY_DEFAULT)
+		    addr2lineCmd << ' ' << "2>/dev/null";
+	  theCommands.push_back(addr2lineCmd.str());
+	}
+
+	// Uncomment to see the command split
+	// for (auto &s : theCommands) {
+	//	   std::cerr << "__COMMAND_________________________________________" << std::endl;
+	//	   std::cerr << s << std::endl;
+	// }
 	
-	//debug
-	if (gblOptions != NULL && gblOptions->outputVerbosity >= MALT_VERBOSITY_VERBOSE)
-		printf("MALT: %s\n",addr2lineCmd.str().c_str());
-	
-	//hide error if silent
-	if (gblOptions != NULL && gblOptions->outputVerbosity <= MALT_VERBOSITY_DEFAULT)
-		 addr2lineCmd << ' ' << "2>/dev/null";
-	
+
+
+	//printf("MALT: %s\n",addr2lineCmd.str().c_str());
```

Comment:
Same if you can remove ?

Reply:
I will remove the commented debug printouts.